cypress/TestCases/FireboltCertification/UserGrants.feature

```diff
@@ -1,5 +1,709 @@
+        When '3rd party app' invokes the 'Firebolt' API to 'expect error localization locality'
+        Then 'Firebolt' platform responds with 'invalid request for localization locality'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for locality'
+        Then 'Firebolt' platform responds with 'false for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+        #Case-2
+        When User 'starts' recording lifecycle history for '3rd party app'
+        And '3rd party app' invokes the 'Firebolt' API to 'expect error localization locality'
+        Then 'Firebolt' platform responds with 'invalid request for localization locality'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'EMPTY_HISTORY'
+
+    # If lifespan value appActive it ask grant once when app is active and it ask again when app is closed and active again.
+    # Case-1: Allowing grant when app is active and validating
+    # Case-2: Closing and launching the app, allowing grant and validating ui is coming again
+    @Usergrants @coreSDK @sdk @transport @notSupported @requiresPlatformImplementation
+    Scenario: UserGrants.Capabilities R*2.2.2.3 - Positive Scenario: Validate Capabilities with granted (pinChallenge) and lifespan appActive
+        Given the environment has been set up for 'userGrants' tests
+        And 3rd party 'certification' app is launched
+        #Case-1
+        And Framework registers 'pinChallenge' test provider
+        And User 'starts' recording lifecycle history for '3rd party app'
+        When '3rd party app' invokes the 'Firebolt' API to 'get device name'
+        Then 'Firebolt' platform responds with 'expected value for device name'
+        When '3rd party app' invokes the 'Firebolt' API to 'check if capabilities is granted for locality'
+        Then 'Firebolt' platform responds with 'true for capabilities granted'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
+        #Case-2
+        When '3rd party app' invokes the 'Firebolt' API to 'close app with user exit'
+        When 3rd party 'certification' app is launched
+        And User set response for 'set pinchallenge correct pin'
+        And User 'starts' recording lifecycle history for '3rd party app'
+        And '3rd party app' invokes the 'Firebolt' API to 'get device name'
+        Then 'Firebolt' platform responds with 'expected value for device name'
+        When User 'stops' recording lifecycle history for '3rd party app'
+        Then User validates lifecycle history for '3rd party app' with 'background:foreground'
```

Comment:
is this TC @notSupported ?

Reply:
No it is not required, removed @notSupported tag